addons/binding/org.openhab.binding.plclogo/src/main/java/org/openhab/binding/plclogo/internal/PLCLogoDataType.java

```diff
@@ -0,0 +1,61 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.plclogo.internal;
+
+import java.util.Calendar;
+
+/**
+ * The {@link PLCLogoDataType} describes data types supported.
+ *
+ * @author Alexander Falkenstern - Initial contribution
+ */
+
+public enum PLCLogoDataType {
+    INVALID,
+    BIT,
+    WORD,
+    DWORD;
+
+    public static int getBytesCount(final PLCLogoDataType type) {
```

Comment:
There are better patterns for this, for example see here https://github.com/openhab/openhab2-addons/blob/master/addons/binding/org.openhab.binding.rfxcom/src/main/java/org/openhab/binding/rfxcom/internal/messages/RFXComBaseMessage.java#L27